examples/gazebo_iris/bob_iris.cc

```diff
@@ -0,0 +1,75 @@
+// BoB robotics includes
+#include "common/main.h"
+#include "gazebo/node.h"
+#include "hid/joystick.h"
+#include "robots/gazebo_uav.h"
+#include "video/gazebocamerainput.h"
+#include "video/display.h"
+
+// Third-party includes
+#include "third_party/units.h"
+
+// Standard C++ includes
+#include <chrono>
+#include <iostream>
+#include <thread>
+#include <cstring>
+
+using namespace BoBRobotics;
+using namespace std::literals;
+using namespace units::literals;
+using namespace BoBRobotics::Video;
+
+int
+bob_main(int argc, char **argv)
+{
+    /************************************Gazebo setup************/
+
+    // Create our node for publishing joystick values
+    gazebo::transport::NodePtr node = Gazebo::getNode();
+
+    /************************************Gazebo setup end************/
+    std::unique_ptr<Display> display;
+    std::unique_ptr<GazeboCameraInput> cam;
+    switch(argc){
+        case 1:
+            std::cout << "Camera disabled.\n";
+            break;
+        case 3: // Initialize gazebo camera if more than 2 arguments are provided (display switch and camera url)
+            std::cout << "Display switch enabled.\n";
+            if(strcmp(argv[1], "-p") == 0) {
+                std::cout << "Using panoramic camera.\n";
+                cam = std::make_unique<GazeboCameraInput>(node, argv[2], true);
+                display = std::make_unique<Display>(*cam, cv::Size(640,320)); //unwrap resolution needs to be supplied
+            }
+            // else if(strcmp(argv[1], "-s") == 0){
+            //     std::cout << "Using simple camera.\n";
+            //     cam = std::make_unique<GazeboCameraInput>(node, argv[2], false);
+            //     display = std::make_unique<Display>(*cam); //unwrap resolution needs to be supplied
+            // }
```

Comment:
I guess we don't need this?

Reply:
Not really. I kept it in case we need to put a simple camera on the drone. Can be removed if there are no such plans.

Reply:
Is there already a simple camera on the drone, i.e. one that came with the model? If so, then you could just uncomment this. Otherwise I'd just remove it.

Reply:
No there isn't a simple camera model on it. Will remove it then